CONFIGURATION.md

```diff
@@ -53,6 +53,11 @@ These environment variables are required for all services.
 | RELAYER_SCREENER_TOKEN | Authorization token for screener service | string |
 | RELAYER_BLOCKED_COUNTRIES | A list of country codes delimited by spaces | string|
 | RELAYER_EXPRESS_TRUST_PROXY | If set to `true`, then `trust proxy` express setting will be enabled. Defaults to `false` | boolean |
+| RELAYER_FEE_MANAGER_TYPE | Fee manager that will be used for fee estimations | FeeManagerType |
+| RELAYER_FEE_SCALING_FACTOR | Scaling factor (in percent) used to adjust fee validation. Defaults to 100. | number |
+| RELAYER_PRICE_FEED_TYPE | Price feed type that will be used for rate conversions. | PriceFeedType |
```

Comment:
I think we need a shortcut feed type for pools with native tokens (e.g. ETH/MATIC/xDAI)

Reply:
Yes

Reply:
Done